<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Client:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="clientBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="userBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>General</string>
    </property>
   </widget>
   <widget class="QComboBox" name="action_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Reset Password</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Delegate Email</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Delegate Calendar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Edit Info</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Group membership</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Security</string>
    </property>
   </widget>
   <widget class="QComboBox" name="secure_box">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Block spam</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Change MFA Info</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Block sign in</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="statusWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="gen_selector">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>19</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sec_selector">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>59</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSync"/>
    <addaction name="actionSync_all"/>
    <addaction name="actionRefresh_token"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionSync">
   <property name="text">
    <string>Sync</string>
   </property>
  </action>
  <action name="actionSync_all">
   <property name="text">
    <string>Sync all</string>
   </property>
  </action>
  <action name="actionRefresh_token">
   <property name="text">
    <string>Refresh token</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
